docs/dashboard/app-templates-gallery.md

```diff
@@ -0,0 +1,53 @@
+---
+title: App Templates Gallery (preview)
+---
+
+Saleor can be extended with apps - and there are several ways how apps are built.
+
+App Templates Gallery is a part of Saleor Cloud Marketplace. As you can see, it doesn't contain exactly "apps",
+but rather templates ready to be spawned on your own instance.
+
+Saleor core is open source and we believe many our users will prefer their code self-hosted and adjusted.
+
+Quick comparison of apps differences:
```

Comment:
```suggestion
Saleor Cloud Marketplace features two types of entities you can use to extend Saleor: templates and apps.

A template is a publicly available foundation you can build upon. It may require an extra setup to work correctly and you need to host it yourself.

An app is an off-the-shelf software, ready to be used immediately. There are two types of apps: 1st party (built and hosted by Saleor) or 3rd party (built by external developers).

Here is a detailed list of differences between them:
```